app/components/templates/Main/Main.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import Nav from '../../organisms/Nav/Nav';
+import Link from '../../../components/atoms/Link/Link.js'
+
+export default class Main extends React.Component {
+  constructor(props) {
+      super(props);
+  }
+
+  render() {
+    return(
+      <div className="app">
+        <Nav />
+        
+        <main>
```

Comment:
Also a few considerations in order to merge this. @Oliphaunte 

* I think we can avoid use excessive styling of the navbar, a list of links should be enough for testing purposes. (ie not hover or color effects. In think in that way we make the project much more reusable, if the user has to change too many things that means the project is not a boilerplate but kind of an app)

* I'd add templates and empty components to test the behavior of the router, now we're getting a 404